tests/dummy/app/templates/application.hbs

```diff
@@ -41,7 +40,7 @@
              {{#submenu-item active=(is-active "demo.virtual-repeat" currentRouteName) onClick=(transition-to "demo.virtual-repeat")}}Virtual Repeat{{/submenu-item}}
           {{/liquid-if}}
         </div>
-        {{#menu-item active=(is-active "forms" currentRouteName) onClick=(transition-to "forms")}}Forms{{/menu-item}}
+        {{#menu-item active=(is-active "forms" currentRouteName) onClick=(transition-to "forms")}}Form{{/menu-item}}
```

Comment:
You've replaced `Forms` with `Form`. What is the reason for that?
We thought of `Forms` to be a general purpose sectionabout "Forms" in ember-paper, not just about paper-form.